stripe-ui-core/src/test/java/com/stripe/android/uicore/elements/PhoneNumberFormatterTest.kt

```diff
@@ -58,7 +60,31 @@ internal class PhoneNumberFormatterTest {
         assertThat(formatter.format("123")).isEqualTo("(123")
         assertThat(formatter.format("1234567")).isEqualTo("(123)-456+7")
         assertThat(formatter.format("123456789012")).isEqualTo("(123)-456+78901!2")
-        assertThat(formatter.format("123456789012456")).isEqualTo("(123)-456+78901!2")
+        // Re-enable this test when maximum length are properly defined for supported countries
+        // assertThat(formatter.format("123456789012456")).isEqualTo("(123)-456+78901!2")
+    }
+
+    @Test
+    fun `Should filter out digits past E164 defined maximum digits`() {
```

Comment:
Can we add another test for a US phone number that's larger than the designated length?